README.md

```diff
@@ -27,143 +27,124 @@ The diagram below illustrates high-level design of each component as well as bas
+```sh
+miden-node init \
+  --config-path  <STORAGE>/miden-node.toml \
+  --genesis-path <STORAGE>/genesis.toml  
+```
 
-3. Monitor container
+which will generate `miden-node.toml` and `genesis.toml` files. The latter controls the accounts that the genesis block will be spawned with and by default includes a basic wallet account and a basic fungible faucet account. You can modify this file to add/remove accounts as desired.
 
-   ```sh
-   docker ps
-   ```
+Next, bootstrap the chain by generating the genesis data:
 
-    After running this command you should see the name of the container `miden-node` being outputted and marked as `Up`.
+```sh
+miden-node make-genesis \
+  --input-path  <STORAGE>/genesis.toml \
+  --output-path <STORAGE>/genesis.dat
+```
 
-### Debian Packages
+which will create `genesis.dat` and an `accounts` directory containing account data based on the `genesis.toml` file.
 
-The debian packages allow for easy install for miden on debian based systems. Note that there are checksums available for the package.
-Current support is for amd64, arm64 support coming soon.
+> [!NOTE]
+> `make-genesis` will take a long time if you're running the production version of `miden-node`, see the tip in the [installation](#install-using-`cargo`) section.
 
-To install the debian package:
+Modify the `miden-node.toml` configuration file such that the `[store]` paths point to our `<STORAGE>` folder:
 
-```sh
-sudo dpkg -i $package_name.deb
+```toml
+[store]
+database_filepath = "<STORAGE>/miden-store.sqlite3"
+genesis_filepath  = "<STORAGE>/genesis.dat"
+blockstore_dir    = "<STORAGE>/blocks"
 ```
 
-Note, when using the debian package to run the `make-genesis` function, you should define the location of your output:
+Finally, configure the node's endpoints to your liking.
 
-```sh
-miden-node make-genesis -i $input_location_for_genesis.toml -o $output_for_genesis.dat_and_accounts
-```
+### Running the node
 
-The debian package has a checksum, you can verify this checksum by downloading the debian package and checksum file to the same directory and running the following command:
+Using the node configuration file created in the previous step, start the node:
 
 ```sh
-sha256sum --check $checksumfile
+miden-node start \
+  --config <STORAGE>/miden-node.toml \
+  node
 ```
 
-Please make sure you have the sha256sum program installed, for most linux operating systems this is already installed. If you wish to install it on your macOS, you can use brew:
+## Updating
 
-```sh
-brew install coreutils
-```
+We currently make no guarantees about backwards compatibility. Updating the node software therefore consists of wiping all existing data and re-installing the node's software again. This includes regenerating the configuration files and genesis block as these formats may have changed. This effectively means every update is a complete reset of the blockchain.
 
-## Testing
```

Comment:
I think it doesn't hurt to keep the "Testing" section.

Reply:
I was worried users would think it applies to testing their deployed node i.e. that its part of the user guide. I'll see if I can make it clear instead.